src/gap-skipper.js

```diff
@@ -0,0 +1,84 @@
+import Ranges from './ranges';
+
+let seekingObject;
+
+const gapSkipper = function() {
+  this.ready(() => {
+    seekingObject = { player: this,
+                      tech: this.tech_,
+                      seeking: false,
+                      consecutiveUpdates: 0,
+                      timer: null,
+                      playerState: null,
+                      lastRecordedTime: null,
```

Comment:
Remove `lastRecordedTime` and `adativeSeeking` as they don't seem to be removed.